.github/workflows/check_scala3_nightly.yml

```diff
@@ -0,0 +1,23 @@
+name: "Check Scala nightly release"
+on:
+  schedule:
+    - cron: 0 5 * * *
+jobs:
+  test_and_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v13
+      - name: "Query maven-central"
+        run: |
+          TODAY_NIGHTLY=$(./coursier complete org.scala-lang:scala3-compiler_3: | grep $(date '+%Y%m%d'))
+          if [ ! -z $TODAY_NIGHTLY ]; then
+            curl \
+              -X POST \
+              -H "Accept: application/vnd.github.v3+json" \
+              -u "dos65:${{ secrets.GH_TOKEN }}" \
+              https://api.github.com/repos/scalameta/metals/actions/workflows/mtags-auto-release.yml/dispatches \
+              -d "{\"ref\": \"main\", \"inputs\": {\"scala_version\": \"$TODAY_NIGHTLY\"}}"
```

Comment:
I think github cli is worth to try in this case
I found https://cli.github.com/manual/gh_workflow_run and https://docs.github.com/en/actions/advanced-guides/using-github-cli-in-workflows

Reply:
oh, cool! thx!

Reply:
changed